nemo_text_processing/g2p/modules.py

```diff
@@ -550,7 +550,13 @@ def __init__(
             mapping_file=mapping_file,
         )
         self.tones = {'1': '#1', '2': '#2', '3': '#3', '4': '#4', '5': '#5'}
+        # pip3 install jieba pyinyin pypinyin-dict
         from pypinyin import lazy_pinyin, Style
+        from pypinyin_dict.pinyin_data import cc_cedict
+        import jieba
```

Comment:
For this PR, `jieba` is hardcoded, i.e. it is enforced to be used for Chinese g2p. Could you please relax the hardcoded segmenter with  an argument? 

You need to add a line under https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/zh/fastpitch_align_22050.yaml#L76
```
    g2p:
      _target_: nemo_text_processing.g2p.modules.ChineseG2p
      phoneme_dict: ${phoneme_dict_path}
      word_segmenter: jieba
```
and define a list of supported segmenter for now and raise exception if not supported. Add `word_segmenter` to instance of g2p.